socorro/external/es/crashstorage.py

```diff
@@ -17,9 +17,27 @@
 from socorro.external.crashstorage_base import Redactor
 
 
-#==============================================================================
+class RawCrashRedactor(Redactor):
+    """Remove some specific keys from a dict. The dict is modified.
+
+    This is a special Redactor used on raw crashes before we send them
+    to our Elasticsearch database. It is used to remove fields that we don't
+    need to store, in order mostly to save some disk space and memory.
+
+    Not that this overwrites the list of forbidden_keys that would be defined
+    through configuration. That list is hard-coded in the __init__ function.
+    """
+    def __init__(self, config):
+        super(RawCrashRedactor, self).__init__(config)
+
+        # Overwrite the list of fields to redact away.
+        self.forbidden_keys = [
+            'StackTraces',
+        ]
```

Comment:
I'm pretty sure you could simplify this by just setting it as a class property. E.g.

```
class RawCrashRedactor(Redactor):
    """docstring here"""
    forbidden_keys = ['StackTraces']
```

That'd work fine and then you don't have to have all the boiler plate for overriding `__init__`.